cs-xml/src/main/java/io/cloudslang/content/xml/services/ApplyXslTransformationService.java

```diff
@@ -0,0 +1,92 @@
+package io.cloudslang.content.xml.services;
+
+import io.cloudslang.content.utils.OutputUtilities;
+import io.cloudslang.content.utils.StringUtilities;
+import io.cloudslang.content.xml.entities.inputs.ApplyXslTransformationInputs;
+import io.cloudslang.content.xml.utils.XmlUtils;
+
+import javax.xml.transform.Result;
+import javax.xml.transform.Source;
+import javax.xml.transform.Templates;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.net.URL;
+import java.util.Map;
+
+
+/**
+ * Created by moldovas on 9/7/2016.
+ */
+
+public class ApplyXslTransformationService {
+    public final Map<String, String> execute(final ApplyXslTransformationInputs applyXslTransformationInputs)
+            throws Exception {
+        final Templates template = getTemplate(applyXslTransformationInputs);
+        final Transformer xmlTransformer = template.newTransformer();
+        xmlTransformer.setOutputProperty("{http://xml.apache.org/xslt}indent-amount", "2");
```

Comment:
Removed property since there was a Warning from Xerces and apparently that`s a documented bug, which doesn`t affect functionality.